README.md

```diff
@@ -17,6 +17,14 @@
 
 The IonosCloud provider gives the ability to deploy and configure resources using the IonosCloud APIs.
 
+---
+### Warning: API Basic Authentication Deprecation Notice
+Effective March 15, 2024, IONOS account holders using 2-Factor Authentication will no longer be able to utilize Basic Authentication for accessing our APIs, SDKs, and all related tools. Token creation and deletion via APIs and ionosCTL will also be restricted.
```

Comment:
will affected users need to regenerate their tokens after this date ?

Reply:
From the documentation, it seems like they need to regenerate their tokens: 
`Can I use existing Basic Authentication tokens after the deprecation date?`
`No, after the deprecation date (March 15, 2024), Basic Authentication tokens will no longer be valid for authorization for 2FA-enabled and forced users. You must switch to using tokens provided by the Token Manager.`

I will update the docs with a reference to the FAQ page.